### ISSUE TITLE
Update required to support new libav

### ISSUE THREAD

**Issue by Psychoteur** [ejurgensen/forked-daapd#178]
Hello,

I've updated my jail in freenas 9.3. I shouldn't since then I've got:

Starting forked_daapd.
/usr/local/lib/libavcodec.so.55: version LIBAVCODEC_55 required by /usr/sbin/forked-daapd not found
/usr/local/etc/rc.d/forked-daapd: WARNING: failed to start forked_daapd

Silly me ;) 

I tried to cheat with ln -s but doesn't work, pfff  

Thanks.

**Comment by ejurgensen** [comment]
I'm not certain what the issue is here? If the library is gone you will need to rebuild forked-daapd. Next time you could configure it with --enable-static, so it doesn't stop working on library upgrades. I haven't tried or tested this, but I think that should do the trick.

**Comment by Psychoteur** [comment]
Hello,
I did rebuilt forked-daapd. Yeah, I should have put --enable-static

I dont't know why it requests/usr/local/lib/libavcodec.so.55. I've got/usr/local/lib/libavcodec.so.56
This is why I tried ln -s libavcodec.so.55 libavcodec.so.55

**Comment by ejurgensen** [comment]
Did you get this sorted out? Seems like it would be a matter of cleaning your build environment so the correct version of libavcodec gets used. There is nothing in the forked-daapd code that selects the version, this is done at build time by the pkgconfig tool.

**Comment by Psychoteur** [comment]
Nope. I didn't. 
Not really the time to play with that at the moment, too much on my plate. 
I tried few things, but nothing works. I even had "zlib missing ..."
In a month, I will try with a new jail. 
And since then, I think freenas with be version 10. 

I've mounted my music with nfs on my mac. That's enough for me right now. 
It reminds me. I've to do a screenshot. 
I don't know if you remember but we had a discussion on how the compilations are seen in Itunes.

So with direct access Itunes sees the compilations as "compilation" in the top left of itunes menu, not in "Various artists" so we can exclude that is as to do with the tags.  

Thanks.
